static.config.js

```diff
@@ -185,6 +185,8 @@ const makeDepartmentPages = async client => {
 };
 
 export default {
+  siteRoot: 'https://alpha.austin.gov',
```

Comment:
This is good to have, although I'm not sure how to call it elsewhere. Is it accessible using withRouteData from react-static?